src/Umbraco.Web.UI.Client/src/packages/core/utils/bytes/bytes.function.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code has been derived from Lee Kelleher's Contentment.
+ * https://github.com/leekelleher/umbraco-contentment/blob/develop/src/Umbraco.Community.Contentment/DataEditors/Bytes/bytes.js
+ * Copyright © 2019 Lee Kelleher.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
```

Comment:
```suggestion
 * SPDX-License-Identifier: MPL-2.0
 * Copyright © 2019 Lee Kelleher.
```

We can simplify the license using the SPDX identifier.